decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -21,12 +21,33 @@ def author_avatar_url
         author&.avatar&.url || ActionController::Base.helpers.asset_path("decidim/default-avatar.svg")
       end
 
-      # Public: Check if the user has voted the proposal
+      # Public: Check if the user has voted the proposal.
       #
-      # Returns Boolean
+      # Returns Boolean.
       def voted_by?(user)
         votes.any? { |vote| vote.author == user }
       end
+
+      # Public: Checks if the organization has given an answer for the proposal.
+      #
+      # Returns Boolean.
+      def answered?
```

Comment:
By design, if it's rejected then it's answered. But in the future this could change, yes.